<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteProfileDialog</class>
 <widget class="QDialog" name="RouteProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>724</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Route Profile Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>702</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="roadTypeGroup">
      <property name="title">
       <string>Road Speeds and Preferences</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="roadTypeList"/>
           </item>
           <item>
            <widget class="QLabel" name="roadTypeLabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>motorway</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="speedLabel">
             <property name="text">
              <string>expected speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="speedSlider">
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="bonusLabel">
             <property name="text">
              <string>bonus (weighting)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="bonusSlider">
             <property name="minimum">
              <number>-200</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="restrictionCheckBox">
             <property name="text">
              <string>obey vehicle access restrictions</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="onewayCheckBox">
             <property name="text">
              <string>obey one-way rules</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <zorder></zorder>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="turnTimeLabel">
        <property name="text">
         <string>turn time in seconds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="turnTime"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="crossTrafficTurnTimeLabel">
        <property name="text">
         <string>cross-traffic turn time in seconds</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="crossTrafficTurnTime"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="trafficLightTimeLabel">
        <property name="text">
         <string>average delay at traffic lights in seconds</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="trafficLightTime"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="uTurnTimeLabel">
        <property name="text">
         <string>u-turn time in seconds</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="uTurnTime"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="tollLabel">
      <property name="text">
       <string>toll road preference</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="tollSlider">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="vehicleTypeGroup">
      <property name="title">
       <string>Vehicle Type</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="vehicleTypeList"/>
       </item>
       <item>
        <widget class="QLabel" name="weightLabel">
         <property name="text">
          <string>vehicle weight (tons)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="vehicleWeightSlider">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>vehicle height (metres)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="vehicleHeightSlider">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Copy from:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyDrive">
        <property name="text">
         <string>Drive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyCycle">
        <property name="text">
         <string>Cycle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyWalk">
        <property name="text">
         <string>Walk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyHike">
        <property name="text">
         <string>Hike</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RouteProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RouteProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
